.gitmodules

```diff
@@ -86,3 +86,6 @@
 	path = third_party/wifi-ism43362/repo
 	url = https://github.com/ATmobica/wifi-ism43362.git
 	branch = master
+[submodule "third_party/mbed-os-cypress-capsense-button"]
+	path = third_party/mbed-os-cypress-capsense-button
```

Comment:
Last change, can you put the submodule into `third_party/mbed-os-cypress-capsense-button/repo` ? 
This also requires to rework https://github.com/ARMmbed/mbed-os-cypress-capsense-button so there is no `repo` folder and its content is at the root.

Reply:
Done